guillotina/contrib/dbusers/__init__.py

```diff
@@ -0,0 +1,31 @@
+from guillotina import configure
+from guillotina.i18n import MessageFactory
+
+_ = MessageFactory("guillotina.contrib.dbusers")
+
+
+app_settings = {
+    "auth_user_identifiers": [
+        "guillotina.contrib.dbusers.users.DBUserIdentifier"
+    ]
+}
+
+configure.permission("guillotina.NotAuthenticated", "")
+configure.permission("guillotina.Authenticated", "")
+configure.permission("guillotina.AddUser", title="Add plone user")
+configure.permission("guillotina.AddGroup", title="Add plone group")
+configure.grant(permission="guillotina.AccessContent", role="guillotina.Anonymous")
+configure.grant(permission="guillotina.NotAuthenticated", role="guillotina.Anonymous")
+configure.grant(permission="guillotina.Authenticated", role="guillotina.Authenticated")
```

Comment:
Everywhere else in the core,`guillotina.Authenticated` refers to the role.

Could this be confusing?

I guess if we would be using different prefixes for roles and permissions, it would help. Something like: `guillotina.permission:AccessContent` and `guillotina.role:Authenticated`

Maybe something worth doing it in the mid term?

Reply:
I think we should see why these are here. Maybe we don't need any of these anymore. Maybe artifacts from old implementation?

Reply:
deleted them for now, as they seem unused... in case we need to add them back, I will add them properly